proposals/csharp-13.0/overload-resolution-priority.md

```diff
@@ -121,7 +121,7 @@ It is an error to apply `OverloadResolutionPriorityAttribute` to the following l
 * Conversion operators
 * Lambdas
 * Local functions
-* Destructors
+* Finalizers
```

Comment:
I made this change to conform to updated spec language. "Destructor" was replaced everywhere with "finalizer".